bot/cogs/free.py

```diff
@@ -72,30 +72,29 @@ class Free(Cog):
         # Display all potentially inactive channels
         # in descending order of inactivity
         if free_channels:
-            embed.description += "**The following channel{0} look{1} free:**\n\n**".format(
-                's' if len(free_channels) > 1 else '',
-                '' if len(free_channels) > 1 else 's'
-            )
-
             # Sort channels in descending order by seconds
             # Get position in list, inactivity, and channel object
             # For each channel, add to embed.description
             sorted_channels = sorted(free_channels, key=itemgetter(0), reverse=True)
-            for i, (inactive, channel) in enumerate(sorted_channels, 1):
+            if len(sorted_channels) > 3:  # display 3 channels max
```

Comment:
This check isn't necessary because when slicing, any end index greater than the end will just return the entire list. So it could be:

```py
for inactive, channel in sorted_channels[:3]:
    ...
```